views/elements/current_and_next.tmpl

```diff
@@ -1,17 +1,32 @@
 {{define "current_and_next"}}
-	{{if .Current}}
-		{{template "current_next" .Current}}
-	{{end}}
-	{{if .Next}}
-		{{template "current_next" .Next}}
+	{{with .CurrentAndNext}}
+
+		<div class="row">
+			{{if .Current}}
+				<div class="col-xs-4 col-sm-4 now-next-live" style="background-image:url('/images/index-now-next-live-thumb.jpeg');"></div>
+				<div class="col-xs-8 col-sm-4 now-next-now">
+					<h2>Now</h2>
+					{{template "current_next" .Current}}
+				</div>
+
+			{{end}}
+			{{if .Next}}
+				<div class="col-xs-12 col-sm-4 now-next-next">
+				 	<h2 class="mobile-col-left">Next</h2>
+				 	<div class="mobile-col-right">
+						{{template "current_next" .Next}}
+					</div>
+				</div>
+			{{end}}
+		</div>
+
 	{{end}}
 {{end}}
 
 {{define "current_next"}}
 
-<p>
-	<a href="{{.Url}}">{{.Title}}</a><br>
-	{{.StartTime.Format "15:04"}} - {{.EndTime.Format "15:04"}}
-</p>
+<h3><a href="{{.Url}}">{{.Title}}</a></h3>
```

Comment:
Also worth noting .Url corresponds to the URL for the time slot not the show, not sure how to get the show URL.

I think we should remove the about this show button and have the show time as a link to the time slot and the show title as a link to the show page. And making the title and show time underline on hover or something

@ChrisTheBaron @LordAro @TomBellis would be interested to see what you guys think about this.